azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_mqtt.py

```diff
@@ -65,6 +65,7 @@ def _execute_op(self, op):
                 username=self.username,
                 ca_cert=self.ca_cert,
                 x509_cert=self.client_cert,
+                websockets=self.pipeline_root.configObj.websockets
```

Comment:
>websockets=self.pipeline_root.configObj.websockets [](start = 16, length = 50)

I thought the op already has the variable websockets when we initialized the op..

Reply:
The op will have access to the variable websockets via the pipeline_root. That's not to say that the MQTT_Transport has access to the websockets parameter, hence why we're passing it in here as a value.